src/SFA.DAS.AdminService.Web.Tests/Controllers/CertificateTests/WhenUserLoadsAmendsRecipientPage.cs

```diff
@@ -2,13 +2,13 @@
 using Microsoft.AspNetCore.Mvc;
 using NUnit.Framework;
 using SFA.DAS.AdminService.Web.Controllers;
-using SFA.DAS.AdminService.Web.ViewModels;
+using SFA.DAS.AdminService.Web.ViewModels.CertificateAmend;
 
 namespace SFA.DAS.AdminService.Web.Tests.Controllers.CertificateTests
 {
     public class WhenUserLoadsAmendsRecipientPage : CertificateAmendQueryBase
     {
-        private IActionResult _result;
+        /*private IActionResult _result;
```

Comment:
is this supposed to be commented out?

Reply:
The controller was removed - the test is now removed.